mongodbatlas/data_federation.go

```diff
@@ -271,3 +288,112 @@ func (s *DataFederationServiceOp) Delete(ctx context.Context, groupID, name stri
 
 	return resp, err
 }
+
+// ConfigureQueryLimit Creates or updates one query limit for one federated database instance.
```

Comment:
```suggestion
// ConfigureQueryLimit updates one query limit for one federated database instance.
```

Reply:
The same PATCH API is used to create and update this entity.